Assets/Scripts/BlockSpawner.cs

```diff
@@ -44,6 +45,44 @@ private void Start()
 
     public void Init()
     {
+        //read json file
+        string blockType = File.ReadAllText(Application.dataPath + "/JSONs/PossibleBlocks.json");
+        var json = JSON.Parse(blockType);
+
+        for (int i = 0; i < 10; ++i)
```

Comment:
We shouldn't use a predetermined end of the loop. Just have the loop continue indefinitely until the end of the JSON is reached.